src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private int age;
+    private String type;
+    private int serialNumber;
+    private int power;
+    private int price;
 
+    @Override
+    protected Engine clone() throws CloneNotSupportedException {
+        return new Engine();
```

Comment:
if you are using clone() method you are expecting to get a new object with all his variables(and their values)